var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -206,6 +214,9 @@ def cmake_args(self):
                 ]
             )
 
+        if spec.satisfies("@4.4.0: +mpi"):
+            args.extend([self.define_from_variant("ENABLE_MPI_F08", "mpi_f08")])
```

Comment:
🤔 Why extend a list in this case? `CMakePackage`'s `define_from_variant` returns a string (https://spack.readthedocs.io/en/latest/spack.build_systems.html#spack.build_systems.cmake.define_from_variant and https://spack.readthedocs.io/en/latest/spack.build_systems.html#spack.build_systems.cmake.define_from_variant) so couldn't you just append?

Reply:
@tldahlgren I guess because I was just copy-pasting what was above it. 😄  Usual case of "That works, so I'll copy it"

But I'll update it.